### PR TITLE
New docs - user guide

### PR DESCRIPTION
**Pull request by vi4m**
Ralph 3.0 more or less complete documentation

### CODE REVIEW

**Comment by fossilet** [comment]
👍

File: docs/user/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+## Ralph quickstart
+
+In this tutorial we will go through:
+- adding new blade server to the Ralph system
+- taking a dc visualization tour
+
+### Adding data center asset
+
+Let's add a new Blade System which will act as a Load balancer system, whe we install haproxy system on it. We want to have it visualized on the dc view, and assign sone supports and licenses to it.
```
Comment (mkurek) [inline docs/user/quickstart.md]: where we install?

Comment (mkurek) [inline docs/user/quickstart.md]: some supports

File: docs/user/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+* service environment 
+
+but wee need to set-up them first. 
+
+#### Adding model
+
+Model helps you organize many same-model devices. You can enter 'Model' field and start typing to search model from existing database models. But don't worry if the list is empty. Adding model is easy without leaving the form. Just click on the + button inside the model field. 
+In the new window just name this model for example: "PowerEdge R620". Choose category from the "three-like" menu. Don't forget to set model type to "data center" - to use it in data center module. You can add back office models as well - switching type field.
+
+Some device in the DC can occupy single position (U) in Rack Visualization. An example could be blade systems, which could store blade servers inside(on the same U position). We use 'slots' field to view them on the DC Visualization.  , which devices are stored in slots, on the same position. To decide how many slots could be presented on the visualization, choose 'layout of front size field'. In our case choose ""
```
Comment (mkurek) [inline docs/user/quickstart.md]: Some device -> A device IMO. Besides that, what is the point of this sentence?

File: docs/user/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+#### Adding Service
+`Service` field groups many devices of the same purpose, together. An example could be - "Internal Load balancing systems", "Client Joe hardware" and so on.
+This field is required to add new asset to the system.
+
+`Service Environment` is a next level of detail about the System - that's the given service environment. Examples: production, testing, development or simple" prod, test, dev. This is very important information to decide how to patch the systems, deploy it handle upgrades, so we use this information all around the Ralph system.
```
Comment (mkurek) [inline docs/user/quickstart.md]: `simple"` -> `simple`

File: docs/user/quickstart.md (hunk 2 above)
Comment (vi4m) [inline docs/user/quickstart.md]: i removed it.

File: docs/user/quickstart.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+### Accessing the edit asset form
+
+If you've played a bit with your beautiful rack, let's go back to the Asset View by clicking on the "Edit asset" in the Rack Inspector.
+
+Here we can explain the rest of non-obvious fields of the Asset.
+
+* `inventory number` -  could be your own, internal id, in many cases, such as: when you import some data from legacy systems.
+* `task url` - could be used in your existing workflow system such as jira, if you use it.
+* `force deprecation` - in some exceptions, you want to force given asset to be deprecated, even if it is still during the deprecation period.
+* TODO: `required support`
```
Comment (mkurek) [inline docs/user/quickstart.md]: `required support` - check if you know that this asset will require vendor support in the future.

File: docs/user/quickstart.md [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+* `parent`: parent object in hierarchy, usally not required. Couple of examples:
+    *  parent of blade server is blade system
+    *  parent of virtual machine is hypervisor
+    *  parent of openstack vm is openstack tenant
+
+**Tips and tricks**
+>You can reduce the numer of fields visible in the settings->permissions, if you don't use some of them.
```
Comment (mkurek) [inline docs/user/quickstart.md]: link to permissions doc (if we have any) will be useful here - if not now then definitely in the future.

Comment (vi4m) [inline docs/user/quickstart.md]: done

File: docs/user/quickstart.md (hunk 4 above)
Comment (vi4m) [inline docs/user/quickstart.md]: done

File: docs/user/quickstart.md [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+
+![add-asset](/img/quickstart-add-asset.png "Add asset")
+
+There are only 3 fields required to add a new server:
+
+* `model`
+* `barcode`(or `serial number` - if the device doesn't have barcode on it)
+* `service environment`
+
+but wee need to set-up them first.
```
Comment (ghost) [inline docs/user/quickstart.md]: `we`? :)

Comment (vi4m) [inline docs/user/quickstart.md]: done

File: docs/user/quickstart.md [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+### Licenses reports
+
+You can analyze in the details usage of licenses, by using one of our reports.
+
+![quickstart-licenses-reports](/img/quickstart-licences-reports.png "Quickstart licenses reports")
+
+
+## Vendor supports
+
+Supports module are similar to the Licenses(it consists of Asset's tab called "Supports", and separate module in the main menu("Supportst")), but the module stores different type of data.
```
Comment (ar4s) [inline docs/user/quickstart.md]: typo: Supportst

**Comment by szok** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**